libs/blocks/src/lib/fast-payment/base/index.tsx

```diff
@@ -10,34 +10,46 @@ const Base: React.FC<FastPaymentProps> = ({
   description,
   link,
   disclaimer,
-  content: { cards = [], cols = 'three', dense },
+  content: { cards = [], cols = 'three', dense, sliderClass },
+  isRtl = false,
```

Comment:
this should something like a global hook, not a prop, please update it.

Reply:
done with removing the `isRtl` props